src/main/kotlin/com/chattriggers/ctjs/api/world/TabList.kt

```diff
@@ -1,73 +1,66 @@
 package com.chattriggers.ctjs.api.world
 
 import com.chattriggers.ctjs.MCTeam
+import com.chattriggers.ctjs.api.CTWrapper
 import com.chattriggers.ctjs.api.client.Client
 import com.chattriggers.ctjs.api.client.Player
+import com.chattriggers.ctjs.api.entity.Team
 import com.chattriggers.ctjs.api.message.TextComponent
+import com.chattriggers.ctjs.internal.mixins.ClientPlayNetworkHandlerAccessor
+import com.chattriggers.ctjs.internal.mixins.MinecraftClientAccessor
+import com.chattriggers.ctjs.internal.mixins.PlayerListEntryAccessor
 import com.chattriggers.ctjs.internal.mixins.PlayerListHudAccessor
 import com.chattriggers.ctjs.internal.utils.asMixin
 import com.google.common.collect.ComparisonChain
 import com.google.common.collect.Ordering
+import com.mojang.authlib.GameProfile
+import gg.essential.elementa.state.BasicState
 import net.minecraft.client.network.PlayerListEntry
 import net.minecraft.scoreboard.ScoreboardDisplaySlot
+import net.minecraft.scoreboard.ScoreboardObjective
 import net.minecraft.text.Text
+import net.minecraft.util.ApiServices
 import net.minecraft.world.GameMode
+import java.util.*
 
 object TabList {
+    private var needsUpdate = true
+    private var tabListNames = mutableListOf<Name>()
     private val playerComparator = Ordering.from(PlayerComparator())
+    internal var customHeader = false
+    internal var customFooter = false
+    private var tabListHeader = TextComponent("")
+    private var tabListFooter = TextComponent("")
 
     @JvmStatic
     fun toMC() = Client.getTabGui()
 
+    @JvmStatic
+    fun getList(): ScoreboardObjective? = Scoreboard.toMC()?.getObjectiveForSlot(ScoreboardDisplaySlot.LIST)
```

Comment:
Maybe I don't understand, but this seems like a strange method to make public. What does it do exactly? At the very least, the method name is bad and should reflect what it actually returns (the only clue to what this does it in the body of the method, which of course isn't available on the docs).

Reply:
Yeah, I just copied what Scoreboard does as it exposes the scoreboard objective in the sidebar slot. I can remove this though.